### DISCUSSION TITLE
How do i get the DateRangePicker's value when uncontrolled?

### DISCUSSION THREAD

**Discussion by dadamssg** [adobe/react-spectrum#4878] [Q&A] [answered]
I'm not sure where to set the input's `name`s.

**Comment by yihuiliao** [comment, marked as answer]
I believe that this should be fixed in the upcoming release with this [pull request](https://github.com/adobe/react-spectrum/pull/4795). If you look at the tests in the PR, you can see some examples of how to set the input's `name`.

**Comment by dadamssg** [comment]
ah okay cool. Would you happen to know if there is any sort of release schedule?

**Comment by yihuiliao** [comment]
We don't follow a strict release schedule but typically every 1-2 months. We are prepping for release currently so you can expect these changes soon but I can't give a specific date.

**Comment by dadamssg** [comment]
ok perfect. thank you!

**Reply by yihuiliao** [reply]
hey just wanted to let you know what we just released v3.29 so the changes in that PR should be there now :))
